build.fsx

```diff
@@ -1,36 +1,65 @@
 #r "packages/FAKE/tools/FakeLib.dll" // include Fake lib
+#r "System.Management"
 
 open System
 open System.Net
 open System.IO
-
+open System.Management
 open Fake
 open Fake.Git
+open Fake.ProcessHelper
 
 let releaseRepo = "https://github.com/guillermooo/sublime-fsharp-package-releases.git"
 
-let sublimePath () =
-  let UnixPaths =
-      [  (Environment.GetEnvironmentVariable("HOME") + "/Library/Application Support/Sublime Text 3")
-         (Environment.GetEnvironmentVariable("HOME") + "/.config/sublime-text-3") ]
-
-  let WindowsPaths =
-    // Non-standard variable. It saves time while developing.
-    [  Environment.GetEnvironmentVariable("SUBLIME_TEXT_DATA")
-       Environment.ExpandEnvironmentVariables(@"%APPDATA%\Sublime Text 3") ]
-
-  let isWindows = (Path.DirectorySeparatorChar = '\\')
-  let searchPaths = if isWindows then WindowsPaths else UnixPaths
-  let directories =
-     searchPaths
-     |> List.filter Directory.Exists
-
-  match directories.Length with
-     | 0 ->
-         trace "No Sublime text 3 installation found"
-         exit 1
-     | _ ->
-         directories.Head
+// build parameters
+let dataDir = getBuildParam "sublimeDir"
+
+ProcessHelper.killCreatedProcesses <- false
+
+let isWindows = (Path.DirectorySeparatorChar = '\\')
+let sublimeDataPath () =
+    let UnixPaths =
+        [  (Environment.GetEnvironmentVariable("HOME") + "/Library/Application Support/Sublime Text 3")
+           (Environment.GetEnvironmentVariable("HOME") + "/.config/sublime-text-3") ]
+
+    let WindowsPaths =
+      // Non-standard variable. It saves time while developing.
+      [  Environment.GetEnvironmentVariable("SUBLIME_TEXT_DATA")
+         Environment.ExpandEnvironmentVariables(@"%APPDATA%\Sublime Text 3") ]
+
+    let searchPaths = if isWindows then WindowsPaths else UnixPaths
+    let directories =
+        searchPaths
+        |> List.filter Directory.Exists
+
+    match directories.Length with
+    | 0 -> trace "No Sublime text 3 installation found"
+           exit 1
+    | _ -> directories.Head
+
+let getSublimeStartArgs () =
+    if not isWindows then
+        let isRunning = getProcessesByName "Sublime Text" |> Seq.length > 0
+        if isRunning then Some("open", "-a \"Sublime Text\"") else None
+    else
+        let query = "SELECT CommandLine FROM Win32_Process WHERE Name LIKE '%sublime_text%'"
```

Comment:
Yup, I've just hit that error because VS seems to set the prefer 32 bit flag by deault.

I suppose using a PowerShell object from System.Management.Automation would work too. But I think the assembly isn't included by default on Windows. I don't know what'd be weirder, WMI or PS. :)

Anyway, this PR looks good. Will merge as soon as I've had a chance to try it.